bluesky_queueserver/manager/profile_ops.py

```diff
@@ -49,10 +49,33 @@ def load_profile_collection(path):
     file_list.sort()  # Sort in alphabetical order
 
     # Load the files into the namespace 'nspace'.
-    nspace = None
+    nspace = {}
+
+    class IPDummy:
+        def __init__(self, user_ns):
+            self.user_ns = user_ns
+
+        def __call__(self):
+            return self
+
+        def register_magics(self, *args, **kwargs):
+            ...
+
+        def magics(
```

Comment:
This seems to be an incomplete method.

Reply:
As I understand it's not a working code, but work in progress. There are some fundamental issues that needs to be resolved before it works.

Reply:
:+1: